src/rules/whitespaceRule.ts

```diff
@@ -40,7 +40,7 @@ export class Rule extends Lint.Rules.AbstractRule {
         description: "Enforces whitespace style conventions.",
         rationale: "Helps maintain a readable, consistent style in your codebase.",
         optionsDescription: Lint.Utils.dedent`
-            Ten arguments may be optionally provided:
+            Eleven arguments may be optionally provided:
```

Comment:
Let's just say "Several" and avoid having to update this in the future

Reply:
Done :)